src/cmds/keys.py

```diff
@@ -144,8 +145,9 @@ def generate_cmd(ctx: click.Context):
 
 
 @keys_cmd.command("show", short_help="Displays all the keys in keychain")
-def show_cmd():
-    show_all_keys()
+@click.option("--show-mnemonic-seed", help="Show the mnemonic seed of the keys", default=False, show_default=True, is_flag=True)
```

Comment:
Maybe the parameter is not the best one, if you have any better option, I'm happy to change it. I haven't added the "short" version because is something that you shouldn't be doing often.